### PR TITLE
Add bin-dist target

### PR DESCRIPTION
**Pull request by JosephSalisbury**

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
RFR @giantswarm/cena 

Note: I don't know where this Jenkins build has appeared from.

File: Makefile [hunk 1]
```diff
@@ -72,3 +72,10 @@ ci-test: $(SOURCE) VERSION .gobuild
 ci-lint: $(SOURCE) VERSION
 	@echo Linting for $(GOOS)/$(GOARCH)
 	$(LINT_COMMAND)
+
+bin-dist: all
+	mkdir -p bin-dist/
+	cp -f README.md bin-dist/
+	cp -f LICENSE bin-dist/
+	cp $(PROJECT) bin-dist/
+	cd bin-dist/ && tar czf $(PROJECT).$(VERSION).tar.gz *
```
Comment (zeisss) [inline Makefile]: do we want to put the current architecture in the upload name?

Comment (JosephSalisbury) [inline Makefile]: Yeah, I like that

**Comment by zeisss** [comment]
LGTM

**Pull request merged**